### PR TITLE
LUN-389: Upgrade tools dependencies

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]

File: tools/git-version.ts [hunk 1]
```diff
@@ -77,28 +81,34 @@ const commitAndTag = async (
     content: contentBase64,
     branch,
     message: commitMessage,
+    // @ts-expect-error Random typescript error: property sha is not available on type { ..., sha: string, ... }
```
Comment (SteffiStuffel) [inline tools/git-version.ts:84]: Can you please explain?

**Review by SteffiStuffel — APPROVED** [review]
Have you tested this?

**Comment by steffenkleinle** [comment]
> Have you tested this?

Not every command but some, yes.

File: tools/git-version.ts (hunk 1 above)
Comment (steffenkleinle) [reply tools/git-version.ts:84]: The types just don't match for some reason. Didn't investigate again but did when I upgraded the dependency for integreat a while back and came to no conclusion. Feel free to try to make this work, would be nice if you'd find the problem.

**Pull request merged**